git.mk

```diff
@@ -0,0 +1,333 @@
+# git.mk, a small Makefile to autogenerate .gitignore files
```

Comment:
auto generate .gitignore files. Many Makefile.am files need it:
```Makefile.am:-include $(top_srcdir)/git.mk```
you can run find to see it:
```$ find -name Makefile.am -exec grep -H git.mk {} \;```